### PR TITLE
[webhooks][fix-docbug][1492189]

### PR DESCRIPTION
**Pull request by Mahesh2516**
Added creating actionable messages section.

### CODE REVIEW

File: msteams-platform/webhooks-and-connectors/how-to/add-outgoing-webhook.md [hunk 1]
```diff
@@ -21,6 +21,37 @@ Webhooks are a great way for Teams to integrate with external apps. A webhook is
...
+- `TextInput` A single-line or multiline text field with an optional length limit.
+- `DateInput` A date selector with an optional time selector.
+- `MultichoiceInput` An enumerated list of choices, offering either a single selection or multiple selections.
+
+`MultichoiceInput` supports a `style` property that controls whether the list initially appears fully expanded. The default value of `style` depends on the value of `isMultiSelect`.
+
+| `isMultiSelect` | `style` default |
+| --- | --- |
+| `false` or not specified | `compact` |
+| `true` | `expanded` |
```
Comment (KirtiPereira) [inline msteams-platform/webhooks-and-connectors/how-to/add-outgoing-webhook.md:45]: This table needs to be worked on. It is not clear what it is conveying. There need to be proper headers for it.

File: msteams-platform/webhooks-and-connectors/how-to/add-outgoing-webhook.md [hunk 2]
```diff
@@ -21,6 +21,37 @@ Webhooks are a great way for Teams to integrate with external apps. A webhook is
...
+
+Specify both `"isMultiSelect": true` and `"style": true` if you want the multi-select list to be displayed in a compact style.
+
+> [!NOTE]
+> * Specifying `compact` for the `style` property in Microsoft Teams is the same as specifying `normal` for the `style` property in Microsoft Outlook.
+> * Webhooks support only Office 365 (message back) cards and adaptive cards.
```
Comment (KirtiPereira) [inline msteams-platform/webhooks-and-connectors/how-to/add-outgoing-webhook.md:51]: Why is message back in brackets?

File: msteams-platform/webhooks-and-connectors/how-to/add-outgoing-webhook.md [hunk 3]
```diff
@@ -21,6 +21,37 @@ Webhooks are a great way for Teams to integrate with external apps. A webhook is
...
+
+For all other details about connector card actions, see **[Actions](/outlook/actionable-messages/card-reference#actions)** in the actionable message card reference.
```
Comment (KirtiPereira) [inline msteams-platform/webhooks-and-connectors/how-to/add-outgoing-webhook.md:53]:
```Suggestion
For all other details about connector card actions, see **[Actions]**(/outlook/actionable-messages/card-reference#actions) in the actionable message card reference.
```

**Review by KirtiPereira — CHANGES_REQUESTED** [review]
Please verify and incorporate the suggestions and comments.

File: msteams-platform/webhooks-and-connectors/how-to/add-outgoing-webhook.md (hunk 2 above)
Comment (Mahesh2516) [reply msteams-platform/webhooks-and-connectors/how-to/add-outgoing-webhook.md]: removed brackets.

**Review by Trinetra-MSFT — DISMISSED** [review]

**Review by Mahesh2516 — COMMENTED** [review]
Please approve

**Review by surbhigupta12 — APPROVED** [review]
Approving the PR.

**Review by KirtiPereira — APPROVED** [review]
Approved

**Pull request merged**